src/util/support.js

```diff
@@ -0,0 +1,3 @@
+const div = document.createElement('div');
+export const shadowDomV0 = !!div.createShadowRoot;
```

Comment:
What about checking if this is a native function? It seems like everyone is messing with prototypes these days.